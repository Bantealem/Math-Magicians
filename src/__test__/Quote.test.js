import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Module/Quote';

describe('snapshot test for Quote Component', () => {
  it('Matches the snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot('Quote');
  });
});
