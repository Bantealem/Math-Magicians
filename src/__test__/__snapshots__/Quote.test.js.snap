// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for Quote Component Matches the snapshot: Quote 1`] = `
<div
  className="quotes"
>
  <h1>
    Quotes
  </h1>
  <p
    className="quote"
  >
    Math is not about numbers, equations, computations or algorithms: it is about understanding.â€“William Paul Thurston
  </p>
</div>
`;
